dgea/plot_dgea.py

```diff
@@ -94,16 +94,30 @@ def plot_volcano():
         
         df_plot = deseq_results.copy()
         df_plot["-log10(padj)"] = -np.log10(df_plot["padj"])
+        min_value = 1e-100
+        df_plot["p-value"] = df_plot["padj"].apply(lambda x: x if x > min_value else f"less than {min_value}")
+
         df_plot["category"] = "Not significant"
         df_plot["gene"] = df_plot.index
-        df_plot.loc[(df_plot["log2FoldChange"] < -lfc) & (df_plot["padj"] < alpha), "category"] = "Downregulated"
-        df_plot.loc[(df_plot["log2FoldChange"] > lfc) & (df_plot["padj"] < alpha), "category"] = "Upregulated"
+        #df_plot.loc[(df_plot["log2FoldChange"] < -lfc) & (df_plot["padj"] < alpha), "category"] = "Downregulated"
+        #df_plot.loc[(df_plot["log2FoldChange"] > lfc) & (df_plot["padj"] < alpha), "category"] = "Upregulated"
+        df_plot.loc[(df_plot["log2FoldChange"] < -lfc) & (df_plot["padj"] < alpha), "category"] = f"High in {_alternative.get()}"
+        df_plot.loc[(df_plot["log2FoldChange"] > lfc) & (df_plot["padj"] < alpha), "category"] = f"High in {_reference.get()}"
 
-        colormap = {"Not significant": "grey", "Downregulated": "blue", "Upregulated": "red"}
+        colormap = {"Not significant": "grey", f"High in {_alternative.get()}": "blue", f"High in {_reference.get()}": "red"}
+        
+        hover_data = {
+            "log2FoldChange": True,
+            "-log10(padj)": True,
```

Comment:
We can remove the `-log10(padj)` since we have the raw p-values now

Reply:
ok